tests/test_tasking.py

```diff
@@ -24,3 +27,37 @@ def test_construct_order_parameters(tasking_mock):
     assert isinstance(order_parameters, dict)
     assert list(order_parameters.keys()) == ["dataProduct", "params"]
     assert order_parameters["params"]["acquisitionMode"] is None
+
+
+def test_get_quotations(tasking_mock):
+    get_quotations = tasking_mock.get_quotations()
+    assert len(get_quotations) == 23
+    workspace_id_filter = "80357ed6-9fa2-403c-9af0-65e4955d4816"
```

Comment:
Nitpick: consider defining `workspace_id_filter` as global variable :)

Reply:
Same, I have changed from the global fixture. Thanks for reminding me. 
DONE!